docs/gettinginvolved/plugins/index.rst

```diff
@@ -12,6 +12,14 @@ The documents listed in the contents below will show you how to implement a bloc
 
 Your storage driver must be a Python 2.7 library providing a class implementing the `flocker.node.agents.blockdevice.IBlockDeviceAPI <https://github.com/ClusterHQ/flocker/blob/master/flocker/node/agents/blockdevice.py>`_ interface.
 
+.. note::
+	If you want your storage driver to support :ref:`storage-profiles`, you can also implement the `flocker.node.agents.blockdevice.IProfiledBlockDeviceAPI <https://github.com/ClusterHQ/flocker/blob/master/flocker/node/agents/blockdevice.py>`_ interface.
+	
+	Please note that you must implement ``bronze``, ``silver``, and ``gold`` profiles.
+	
+	If you do choose to implement profiles, it would be great to :ref:`hear from you <talk-to-us>` about what you think of both profiles in general, and of the interface.
+	We are looking to extend the profiles functionality, and would love feedback from driver writers.
```

Comment:
we tend to use the (oxford comma)[https://en.wikipedia.org/wiki/Serial_comma] in most cases in the docs - its mainly for clarification, especially for non-english speaking users